src/lib.rs

```diff
@@ -28,12 +28,18 @@ The following code recursively iterates over the directory given and prints
 the path for each entry:
 
 ```rust,no_run
-use walkdir::WalkDir;
+use walkdir::{Error, WalkDir};
```

Comment:
i would suggest splitting the `use` here into two lines and biding the `Error` line as it should be part of the hidden error  boilerplate. 

this  concerns all examples below.